## PV Solar

| Source || Manufacturing || Installation || O&M |
| --- || --- || --- || --- |
| || Job years/MW || Job years/MW || Jobs/MW |
| [1] || 6.7 || 13.0 ||0.7|
| [2] || -- || -- ||1.06|
| [3] || 18.8|| 11.2 ||0.3|
| [4] || 10.3 || 6.6 ||0.2|
| Average || 11.9 || 10.3 ||0.6|
| Median || 10.3 || 11.2 ||0.5|

## Wind

| Source || Manufacturing || Installation || O&M |
| --- || --- || --- || --- |
| || Job years/MW || Job years/MW || Jobs/MW |
| [1] || 4.7 || 3.2 ||0.3|
| [2] ||  -- || --  ||0.05|
| [3] || 4.0 || 2.0 ||0.3|
| [4] || 7.5 || 2.5 ||0.4|
| Average || 5.4 || 2.6 ||0.6|
| Median  || 4.7 || 2.5 ||0.3|

## Nuclear

<img src = 'img/Nuclear_Labor_Years_FCoEP_Table9_2.png'>

In [17]:
'''Table 9.2 does not provide a breakdown of manufacturing vs installation so 
we will calculate it ourself'''
Manufacturing = (316+1994+2244+729+1341+3)/1147
print('Manufacturing = '+str(round(Manufacturing,1)))
Installation = 12.1-Manufacturing
print('Installation = ' + str(round(Installation,1)))
#These are the numbers in the following table for source 2

Manufacturing = 5.8
Installation = 6.3


Source 1 Gives 0.95 jobs/MW, multiply by 10 years decommissioning to 
get 9.5 job years/MW.

Source 2 says 5000 labour-years for decommissioning a 1000 MWe reactor so divide 5000 by 1000 to get 5 job years/MW.

#### Nuclear

| Source || Manufacturing || Installation || O&M || Decommissioning |
| --- || --- || --- || --- ||---|
| || Job years/MW || Job years/MW || Jobs/MW || Job years/MW|
| [1] || 1.3 || 11.8 ||0.6|| 9.5|
| [2] || 5.8 || 6.3  ||0.5|| 5.0|
| Average/Median || 3.6 || 9.1 ||0.6||7.3|

## Coal

| Source || Manufacturing || Installation || O&M |
| --- || --- || --- || --- |
| || Job years/MW || Job years/MW || Jobs/MW |
| [1] || 5.4 || 11.2 ||0.14|
| [2] || -- || -- ||0.19|
| Average/Median || 5.4 || 11.2 ||0.17|

## Natural Gas

| Source || Manufacturing || Installation || O&M |
| --- || --- || --- || --- |
| || Job years/MW || Job years/MW || Jobs/MW |
| [1] || 0.93 || 1.3 ||0.14|

## Sources

<a href="https://opus.lib.uts.edu.au/handle/10453/43718" >Calculating Global Energy Sector Jobs</a>[1]

<a href="https://www.oecd-nea.org/jcms/pl_14998/the-full-costs-of-electricity-provision?details=true" >The Full Costs of Electricity Provision</a>[2]

<a href="https://www.sciencedirect.com/science/article/abs/pii/S1364032115000118" >Employment factors for wind and solar energy technologies: A literature review</a>[3]

<a href="https://www.sciencedirect.com/science/article/abs/pii/S0360544215011378" >Employment effects of renewable electricity deployment. A novel methodology</a>[4]